splice_spark2/src/main/scala/com/splicemachine/spark2/splicemachine/SplicemachineContext.scala

```diff
@@ -589,13 +590,13 @@ class SplicemachineContext(options: Map[String, String]) extends Serializable {
       val sqlText = insertSql(topicInfo)
       debug(s"SMC.inss sql $sqlText")
       
-      trace( s"SMC.inss topicCount preex ${KafkaUtils.messageCount(kafkaServers, topicName)}")
+      //trace( s"SMC.inss topicCount preex ${KafkaUtils.messageCount(kafkaServers, topicName)}")
 
```

Comment:
Do you want to remove commented out logging?

Reply:
These logs were useful during some of the performance test runs.  It may be best to eventually add a "diagnostic" flag which would activate these logs.  I currently don't want them to call KafkaUtils.messageCount() unnecessarily.